docs/content/getting-started/first-app/snippets/app-with-redis-snippets.bicep

```
@@ -0,0 +1,41 @@
+import radius as radius
+param application string
+param environment string
+
+resource demo 'Applications.Core/containers@2022-03-15-privatepreview' = {
+  name: 'demo'
+  location: 'global'
+  properties: {
+    application: application
+    container: {
+      image: 'radius.azurecr.io/tutorial/webapp:edge'
+      ports: {
+        web: {
+          containerPort: 3000
+        }
+      }
+    }
+    //CONNECTION
+    connections: {
+      redis: {
+        source: db.id
+      }
+    }
+    //CONNECTION
+  }
+}
+
+//REDIS
+resource db 'Applications.Link/redisCaches@2022-03-15-privatepreview' = {
+  name: 'db'
+  location: 'global'
+  properties: {
+    application: application
+    environment: environment
+    mode: 'recipe'
+    recipe: {
+      name: 'redis'
```

I'm assuming this is meant to use dev recipe? If so, the format is link-provider https://github.com/project-radius/radius/blob/main/pkg/corerp/frontend/controller/environments/createorupdateenvironment.go#L138

```suggestion
      name: 'redis-kubernetes'
```

---

Thanks :)